CONTRIBUTING.md

```
@@ -52,6 +52,100 @@ Once we have all the requirements properly installed and configured, we can buil
 ./mvnw clean install
 ```
 
+## IntelliJ IDEA Setup
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)